conf/application.conf

```diff
@@ -21,6 +21,7 @@ appName = pension-scheme-return
 
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
Some libraries still use HttpClient, we'll try to upgrade and remove `HttpClientModule` in a separate branch/PR.

Reply:
Resolved in part2 PR: https://github.com/hmrc/pension-scheme-return/pull/105